fog-azure-rm.gemspec

```diff
@@ -22,8 +22,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest', '~> 5.8.4'
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'codeclimate-test-reporter' , '~> 1.0.0'
-  spec.add_dependency 'fog-core', '~> 1.43'
-  spec.add_dependency 'fog-json', '~> 1.0.2'
+  spec.add_dependency 'fog-core'
+  spec.add_dependency 'fog-json'
```

Comment:
@maham-nazir-confiz I have removed the version dependency here. Could you please take a look?

Reply:
I'll get back to you after reviewing the changes.

Regards

Reply:
@maham-nazir-confiz Any updates on this?